### DISCUSSION TITLE
Boost information as icons

### DISCUSSION THREAD

**Discussion by RALLITTELIJA** [runelite/runelite#15130] [Ideas]
Would it be possible to add boosts information as icons? This would be really convenient for players with small clients who don't want them to be as infoboxes as they can get really stacked/messy. The other available option with text is too large and it covers a lot of space from the client itself. There is a reference picture and the boosts are right next to the inventory. Buff/debuff would be in the bottom with them too!
![Valuable drop Hydra leather (4,054,444 coins) 2021-11-07_14-16-37](https://user-images.githubusercontent.com/107722903/174400333-5f26cee6-ea2b-46c2-be83-d71699e325b6.png)

**Comment by Alexsuperfly** [comment]
this is exactly what infoboxes do

**Reply by RALLITTELIJA** [reply]
Those boosts will be grouped with other infoboxes and it can become really cluttered with other not so important stuff.
![799C4984-5F91-42C4-9C09-6A7E4ED08EB7](https://user-images.githubusercontent.com/107722903/174420454-738ff4e5-1a8d-47eb-9997-ae40fb4b9fd8.png)

**Reply by Alexsuperfly** [reply]
so then detach the boost infoboxes and place them where ever you want them

**Reply by RALLITTELIJA** [reply]
If there is a way to keep them as infoboxes and detach them from the group I'm not aware of that. Are you talking about the other setting which makes the boosts appear as text? If so, that overlay takes up too much space from screen and customization options are close to none. If I wanted to make the window smaller without text overlapping I'd need to make every overlay smaller and that is not optimal.

**Reply by Alexsuperfly** [reply]
hold shift and right click them

**Reply by RALLITTELIJA** [reply]
Not the prettiest and again can't customize boosts without fucking up all the other infoboxes but will do for now. Consider this one!

**Reply by wulfeh123** [reply]
> Not the prettiest and again can't customize boosts without fucking up all the other infoboxes but will do for now. Consider this one!

Also agree that the infoboxes are pretty ugly compared to the version in the screenshot. Would love if icons were an option in the base Ruenlite version of the plugin.

**Comment by Adam-** [comment]
Took a stab at this, this is all there is to it? No other functionality?
![image](https://user-images.githubusercontent.com/309739/174465200-48a97748-ecb1-4856-8177-536f70b1d4f0.png)

**Reply by RALLITTELIJA** [reply]
Basically yeah, also buff/debuff counter would be grouped with them.
![OLrYRE0](https://user-images.githubusercontent.com/107722903/174465449-ba57b5ee-628f-418b-b106-37506d03ebc4.png)

**Reply by Adam-** [reply]
Buff/debuff counter are two different things. You want both?

**Reply by RALLITTELIJA** [reply]
The player could decide if he wants to have them both, only one of them or neither one like in the boosts information plugin.

**Reply by RALLITTELIJA** [reply]
I assume this would be added into boosts information plugin and there are those customization options already available.

**Comment by Adam-** [comment]
Okay I've added this for the next update
